src/bridge/cockpitinternalmetrics.c

```diff
@@ -242,6 +240,11 @@ cockpit_internal_metrics_sample (CockpitSamples *samples,
 {
   CockpitInternalMetrics *self = COCKPIT_INTERNAL_METRICS (samples);
 
+  if (self->omit_instances)
```

Comment:
OK, added locally. Waiting a bit with the push in case you have some other feedback, to not unnecessarily run CI.